iOS/CustomFontNavigationPageRenderer.cs

```diff
@@ -14,8 +17,8 @@ public CustomFontNavigationPageRenderer()
 		{
 		}
 
-		const string customFontName = "FontAwesome.ttf";
```

Comment:
using this font was resulting in returning a null from UIFont.FromName(), so I replaced it with a known font name.